docs/reference/release.md

```diff
@@ -43,60 +43,16 @@ When backing up release artifacts, you may use the following fields in your filt
 expressions. These fields are accessed using the `release.<field>` syntax, for example
 `release.prerelease` to determine if a release is a pre-release.
 
-```json
-{
```

Comment:
Any strong reason to move away from using the JSON format? I figured this might make it a bit easier to maintain and avoid confusion about what the values look like whilst still remaining mostly readable for the target audience.

No major concerns from my wife either way, especially since you already put in the work.

Reply:
no, no strong reason at all. 
I just copied the tables from the README.md of the project there when I created the gist reference and thought to myself "hmm, what If I just used the Markdown table from the README.md here instead of having to format it as JSON?" and then I looked at the table and realized "hmm, but now release and repository are different from gist..."

I have no preference on either one. I think the JSON works just fine